be/src/vec/exec/scan/vfile_scanner.cpp

```diff
@@ -580,6 +580,8 @@ Status VFileScanner::_get_next_reader() {
             _cur_reader.reset(new CsvReader(_state, _profile, &_counter, _params, range,
                                             _file_slot_descs, _io_ctx.get()));
             init_status = ((CsvReader*)(_cur_reader.get()))->init_reader(_is_load);
+            _state->set_support_read_bytes(
```

Comment:
maybe we should add scan_bytes feature to all scan node. For example, broker load, insert into select, stream load and etc.